ci/platform-independence.yml

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+steps:
+  - bash: ci/dev-env-install.sh
+    displayName: 'Build/Install the Developer Environment'
+  - bash: ci/configure-bazel.sh
+    displayName: 'Configure Bazel for root workspace'
+    env:
+      IS_FORK: $(System.PullRequest.IsFork)
+      # to upload to the bazel cache
+      GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
+  - task: DownloadPipelineArtifact@0
+    inputs:
+      artifactName: 'platform-independence-dar'
+      targetPath: $(Build.SourcesDirectory)/platform-independence.dar
+  - bash: |
+      bazel build //compiler/damlc/tests:platform-independence.dar -o platform-independence-linux.dar
+    displayName: 'Rebuild platform-independence.dar'
+  - bash: |
```

Every `bash` task at the YAML level is a separate Bash process so you need to activate dev-env again here:
```suggestion
  - bash: |
      set -euo pipefail
      eval "$(dev-env/bin/dade-assist)"
```